widgets/SuccessFactors-connector.json

```diff
@@ -34,5 +96,79 @@
       ]
     }
   ],
+  "filters":[
+    {
+      "name": "basicAuth",
+      "condition": {
+        "property": "authType",
+        "operator": "equal to",
+        "value": "basicAuth"
```

Comment:
`authType` will be `null` in existing pipelines since it is a newly added config.

We should default to `basicAuth` when `authType` is also `null`.